test/associative/types/testNilableBorrowed.chpl

```diff
@@ -1,15 +0,0 @@
-class T {
-  var value = 0;
-}
-
-var D: domain(int);
-
-D += 1;
-
-var A: [D] borrowed T?;
-var a = new T();
-var b = a.borrow();
-
-A[1] = b;
-
-assert(A.size == 1);
```

Comment:
How did this test behave after your compiler changes?  It kinda seems like something we want to support, so it'd be nice if the test would stick around.

Reply:
This one still compiles cleanly, I can reinstate it